### PR TITLE
Add TargetQueryValue to KEDA Autoscaler

### PR DESCRIPTION
**Pull request by dimberman**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
CC: @turbaszek ?

File: chart/templates/workers/worker-kedaautoscaler.yaml [hunk 1]
```diff
@@ -42,6 +42,7 @@ spec:
   triggers:
     - type: postgresql
       metadata:
+        targetQueryValue: "1"
         connection: AIRFLOW_CONN_AIRFLOW_DB
         query: "SELECT ceil(COUNT(*)::decimal / 16) FROM task_instance WHERE state='running' OR state='queued'"
```
Comment (turbaszek) [inline chart/templates/workers/worker-kedaautoscaler.yaml:47]: Btw. can we template the `16` value?

**Review by turbaszek — APPROVED** [review]
LGTM me 👌

File: chart/templates/workers/worker-kedaautoscaler.yaml (hunk 1 above)
Comment (dimberman) [reply chart/templates/workers/worker-kedaautoscaler.yaml:47]: Yes we can but let's do a different PR for that

**Review by kaxil — COMMENTED** [review]
LGTM :)

**Pull request merged**